Automattic/jetpack
# PWA: Add skeleton module with manifest

Factors out manifest functionality from #7963.

To provide Progressive Web Apps (PWA) functionality, we need to provide a manifest. This manifest file contains information about the website (application) such as its name, icons, etc.

In the future, we will also use this file for other purposes, but for now, some basic descriptive information will work.

This PR also adds the necessary PWA skeleton module and some basic module structure.

To test:

- Checkout PR
- In `wp shell`, `wp jetpack module activate pwa`
- Load front-end of site
- Ensure that there's a tag with `rel="manifest"`
- Load up the manifest URL and ensure it shows proper values
- Run a Chrome audit and ensure it passes any manifest tests
